README.md

```diff
@@ -19,19 +19,14 @@
 
 ## About
 
-> **Note**: this project is still relatively young and active development with
-> large changes is still happening. If you're considering depending on this at
-> this time it's recommended to reach out to the authors on [zulip] and get more
```

Comment:
keeping a pointer somewhere to zulip as a place to get help is usefull to new devs I think